README.md

```diff
@@ -31,6 +31,28 @@ Navigate to the directory where you want to create your new project. Once you're
 
 After running this command, a new project based on the template will be generated, complete with the folder structure, configurations, and code necessary to start developing your application.
 
+### Step 4 (optional): Dockerfile
+
+If you want to use the template Dockerfile for building your .NET app then you can must update the `PROJECT_NAME` and `REPO_ORIGIN` build arguments in the file
+
+**Dockerfile** - Option 1
+
+```diff
+-ARG PROJECT_NAME="DfE.DomainDrivenDesignTemplate.Api"
++ARG PROJECT_NAME="My.NewProject.Api"
+-ARG REPO_ORIGIN="https://github.com/DFE-Digital/rsd-ddd-clean-architecture"
```

Comment:
Can we please change all the names to "DfE.DomainDrivenDesignTemplate.*" the template automatically replaces any "DfE.DomainDrivenDesignTemplate" with the user's new name